styles/components/forms/slider/slider.json

```diff
@@ -1,9 +1,5 @@
 {
-  "elements": [
-    {
-      "name": "demo",
-      "min": 0,
-      "max": 10
-    }
-  ]
+  "min": 0,
+  "max": 10,
+  "input": "<input type=\"range\" min=\"0\" max=\"10\" value=\"5\">"
```

Comment:
@lauriii Does the comment above seem valid? I think this is our only component that has a .json file though.